app/errors.py

```diff
@@ -0,0 +1,19 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    # def __init__(self, message: str = "Visitor is not vaccinated") -> None:
+    #     super().__init__(message)
+    def __str__(self) -> str:
+        return "Visitor is not vaccinated"
+
+
+class OutdatedVaccineError(VaccineError):
+    def __str__(self) -> str:
+        return "All visitors must wear masks"

```

Comment:
The `OutdatedVaccineError` class returns a message that is incorrect. It should return 'Vaccine is expired' instead of 'All visitors must wear masks'.